src/applications/pact-act/tests/cypress/helpers.js

```diff
@@ -39,7 +39,7 @@ export const selectRadio = (selector, index) =>
   cy
     .findByTestId(selector)
     .should('exist')
-    .get('va-radio-option')
+    .get('[data-testid=va-radio-option]')
```

Comment:
This should be `.get('[data-testid="va-radio-option"]')` - this passes without the double quotes around `va-radio-option`?

Reply:
Good catch. It was passing but I will make the change

Reply:
Poked around a little. There are places where the double quotes aren't necessary. Seems like Cypress is one of them, but also in raw HTML (since HTML 2.0) 🤯 TIL